entrypoint.sh

```diff
@@ -195,7 +195,7 @@ do
     # quite likely to succeed. Not sure if more than 1 loop iteration will
     # ever be hit in the real world.
     set -x
-    git pull # error out if pull fails (errexit)
+    git pull origin "${DATA_BRANCH_NAME}" # error out if pull fails (errexit)
```

Comment:
Hey! Sorry for replying only now. Can you maybe describe in the pull request in which situation this patch is important?